### PR TITLE
deleteFile mutation

### PR DESCRIPTION
**Pull request by parthc2fo**
I have used raw query here to get the deleted data as typrORM is not supported something like `findOneAndDelete` query

### CODE REVIEW

File: src/file/file.service.spec.ts [hunk 1]
```diff
@@ -49,44 +68,91 @@ describe("FileService", () => {
...
+
+      const FileInput: FileUpload = {
+        filename: "git-cheatsheet.pdf",
+        mimetype: "application/pdf",
+        encoding: "",
+        createReadStream: jest.fn(),
+      };
+
+      jest.spyOn(fileRepository, "create").mockReturnValue({
+        id: "3dd6d207-066c-42e5-bbed-f40ce7355941",
```
Comment (yadavpradeep786) [inline src/file/file.service.spec.ts]: All mock data can be moved to mocked file

**Review by yadavpradeep786 — CHANGES_REQUESTED** [review]
All mock data can be moved to mocked file

File: src/file/file.service.spec.ts (hunk 1 above)
Comment (parthc2fo) [reply src/file/file.service.spec.ts]: Sure

**Comment by parthc2fo** [comment]
I have moved all the mocked data to a separate mock folder for upload and delete mutation

cc - @yadavpradeep786 @m-parul

File: src/file/file.service.spec.ts (hunk 1 above)
Comment (parthc2fo) [reply src/file/file.service.spec.ts]: commits 
- [9fb2967](https://github.com/parthc2fo/upload-download-thing/pull/5/commits/9fb29679453e2741ad232427ec43855b199b24c4)
- [777e34a](https://github.com/parthc2fo/upload-download-thing/pull/5/commits/777e34ab79c8d1caafa285ded879152b5cf8cb36)

**Review by yadavpradeep786 — APPROVED** [review]
LGTM

**Comment by parthc2fo** [comment]
I am merging this PR

**Pull request merged**